cfg/python/defaultconfig.py

```diff
@@ -176,7 +176,7 @@ def mc(cfg, **kwargs):
         'x1','x2','qScale',  # qScale taken from hard process saved in MC
         'numberGeneratedEventsWeight','crossSectionPerEventWeight','generatorWeight','puWeight',
         ]
-    
+
     #lheWeightNames = ['nominal','isrDefup','isrDefdown','fsrDefup','fsrDefdown']
```

Comment:
This can be put in, since the default behaviour for the GenWeightProducer is available.

Reply:
I only see a whitespace-related change here. Could you clarify what you mean?

Reply:
I mean the commented `lheWeightNames` part. Since we implemented a behaviour for default genWeights in #61, we can add the according lines to the default config.